RELEASE-NOTES.md

```diff
@@ -4,6 +4,7 @@
 ## Deprecations
 
 ## New additions
+ * Add `snow notebook create` command
```

Comment:
Please remove it from release notes, notebooks are not yet in public preview.

Reply:
✅